packages/thirdweb/src/extensions/prebuilts/deploy-pack.test.ts

```diff
@@ -1,14 +1,14 @@
 import { describe, expect, it } from "vitest";
 import { ANVIL_CHAIN } from "~test/chains.js";
 import { TEST_CLIENT } from "~test/test-clients.js";
-import { TEST_ACCOUNT_A } from "~test/test-wallets.js";
+import { TEST_ACCOUNT_D } from "~test/test-wallets.js";
 import { isAddress } from "../../utils/address.js";
 import { deployPackContract } from "./deploy-pack.js";
 
 describe.runIf(process.env.TW_SECRET_KEY)("deploy-pack contract", () => {
   it("should deploy Pack contract", async () => {
     const address = await deployPackContract({
-      account: TEST_ACCOUNT_A,
+      account: TEST_ACCOUNT_D,
```

Comment:
why did this need to change? seems unrelated?

Reply:
Reverted!

Reply:
@jnsdls this should be ready (I accidentally cancel the review)